source/upgrade.txt

```diff
@@ -289,18 +289,18 @@ This driver version introduces the following breaking changes:
   code using these methods must be recompiled. This change does not require any
   changes to source code.
   
-- Removes the following methods and types related to the 
-  `StreamFactory
-  <https://mongodb.github.io/mongo-java-driver/4.11/apidocs/mongodb-driver-core/com/mongodb/connection/StreamFactory.html>`__
-
-- Removes the following methods and types related to the 
+- Removes the following methods and types related to the
   `Stream
   <https://mongodb.github.io/mongo-java-driver/4.11/apidocs/mongodb-driver-core/com/mongodb/connection/Stream.html>`__
   interface:
```

Comment:
This duplication probably to have happened as a result of (presumably) rebasing and manually resolving non-trivial conflicts in https://github.com/mongodb/docs-java/pull/513 after the review had been completed.